### PR TITLE
PTRENG-4192 - updated and use shared module

### PR DESCRIPTION
**Pull request by chb0github**

### CODE REVIEW

File: pkg/artifactory/resource/security/resource_artifactory_scoped_token.go
Comment (alexhung) [inline pkg/artifactory/resource/security/resource_artifactory_scoped_token.go:105]: Are you sure we don't need `\` to escape the `*`? This is for the literal `*` character in the scope value, not regex.

**Review by alexhung — APPROVED** [review]
LGTM. Just one question about the regex for scoped token.

File: pkg/artifactory/resource/security/resource_artifactory_scoped_token.go
Comment (chb0github) [reply pkg/artifactory/resource/security/resource_artifactory_scoped_token.go:105]: I can add it back in - IJ flagged it as unnecessary

Comment (chb0github) [reply pkg/artifactory/resource/security/resource_artifactory_scoped_token.go:105]: Tests pass - let's leave it.

**Comment by chb0github** [comment]
I updated the changelog - let's make sure quality gates pass then I'll merge

**Comment by chb0github** [comment]
@alexhung - I don't see failing tests - it says need code owner review and 1 approval

**Review by alexhung — APPROVED** [review]

**Pull request merged**
